src/Controls/src/Core/Handlers/Items/StructuredItemsViewHandler.Windows.cs

```diff
@@ -245,20 +252,35 @@ static WStyle GetItemContainerStyle(GridItemsLayout layout)
 
 			var style = new WStyle(typeof(GridViewItem));
 
+			if (_gridViewItemStyle is not null)
+			{
+				style.BasedOn = _gridViewItemStyle;
+			}
+
 			style.Setters.Add(new WSetter(FrameworkElement.MarginProperty, margin));
 			style.Setters.Add(new WSetter(Control.PaddingProperty, WinUIHelpers.CreateThickness(0)));
 			style.Setters.Add(new WSetter(Control.HorizontalContentAlignmentProperty, HorizontalAlignment.Stretch));
 
 			return style;
 		}
 
+		static WStyle GetDefaultStyle(string resourceKey)
+		{
+			return Microsoft.UI.Xaml.Application.Current.Resources[resourceKey] as WStyle;
```

Comment:
Could we include a fallback to default styles in case cannot find the passed `resourceKey`?

Reply:
Checked, is fine.